segmentation_labeling_app/transforms/transformations.py

```diff
@@ -0,0 +1,212 @@
+from pathlib import Path
+import random
+from typing import Union, Tuple
+import os.path
+import math
+import copy
+
+import h5py
+import cv2
+import numpy as np
+import imageio_ffmpeg as mpg
+
+
+def downsample_h5_video(video_path: Union[str, Path], input_fps: int,
+                     output_fps: int, strategy: str = 'random') -> np.ndarray:
+    """
+    Function to down-sample an array stored in h5 format to a desired FPS.
+    Follows the strategy specified in the function call, current strategies are
+    (random, maximum, average). Random takes a random sample of one frame from
+    window, maximum takes the maximum frame in window, average takes the mean
+    of window.
+    Args:
+        video_path: The path to the input video in str or Path format
+        input_fps: The FPS or Hz of the input video
+        output_fps: The desired output FPS or Hz
+        strategy: The down-sampling startegy to follow
+
+    Returns:
+        new_video: A new video down-sampled to desired FPS with selected
```

Comment:
this should document a type. Also the Args above.

Reply:
if it's documented in the function signature the type isn't to be specified in pep I believe